res/TensorFlowLiteRecipes/Add_008/test.recipe

```diff
@@ -0,0 +1,32 @@
+operand {
+  name: "ifm1"
+  type: FLOAT32
+  shape { dim: 4 dim: 2 dim: 3 }
+}
+operand {
+  name: "ifm2"
+  type: FLOAT32
+  shape { dim: 1 dim: 2 dim: 3 }
```

Comment:
Broadcast to dim 0, with rank 3

Reply:
FYI, in `ONE/compiler/luci` and related, `rank` is used for number of dimensions,
so here IIUC `rank 3` can be expressed as `size 3`